src/main/java/mekanism/common/Mekanism.java

```diff
@@ -544,6 +544,8 @@ public static void addRecipes() {
                   new GasStack(MekanismFluids.Oxygen, 10), new ItemStack(MekanismItems.Polyethene), new GasStack(MekanismFluids.Oxygen, 5), 1000, 60);
             RecipeHandler.addPRCRecipe(new ItemStack(MekanismItems.Substrate), new FluidStack(FluidRegistry.WATER, 200), new GasStack(MekanismFluids.Ethene, 100),
                   new ItemStack(MekanismItems.Substrate, 8), new GasStack(MekanismFluids.Oxygen, 10), 200, 400);
+            RecipeHandler.addPRCRecipe(new ItemStack(Items.COAL, 1, OreDictionary.WILDCARD_VALUE), new FluidStack(FluidRegistry.WATER, 100), new GasStack(MekanismFluids.Oxygen, 100),
```

Comment:
I don't think this one is necessary, since you're doing it by oredict later?

Reply:
It is as i only add Coal Block and dust later with OreDict. but i can move/change that to OreDict if you think that would be better.

Reply:
so just tell me if you think it should be coal OreDict instead of just Coal.

Reply:
Ah, right you are. I got mixed up with dust. Leave it as it is, the recipes will get redone later anyway when we fix it to be more flexible